www/js/desktop_scripts.js

```diff
@@ -82,39 +82,50 @@ module.exports = {
     }
     isOnline().then((online) => {
       if (online) {
-        request('https://banidb.com/databases/sttmdesktop.md5', (error, response, newestDBHash) => {
+        request('https://banidb.com/databases/sttmdesktop.realm.md5', (error, response, newestDBHash) => {
           if (!error && response.statusCode === 200) {
             const curDBHash = store.get('curDBHash');
             if (force || curDBHash !== newestDBHash) {
-              const dbZip = path.resolve(userDataPath, 'sttmdesktop.zip');
-              progress(request('https://banidb.com/databases/sttmdesktop.zip'))
+              const dbCompressed = path.resolve(userDataPath, 'sttmdesktop.realm.bz2');
+              progress(request('https://banidb.com/databases/sttmdesktop.realm.bz2'))
                 .on('progress', (state) => {
                   const win = remote.getCurrentWindow();
                   win.setProgressBar(state.percent);
                   global.core.search.updateDLProgress(state);
                 })
                 .on('end', () => {
-                  const zip = new AdmZip(dbZip);
-                  zip.extractEntryTo('sttmdesktop.db', userDataPath, true, true);
-                  module.exports.initDB();
-                  store.set('curDBHash', newestDBHash);
-                  fs.unlinkSync(dbZip);
-                  // Delete pre-4.0 DB
-                  const oldDB = path.resolve(userDataPath, 'data.db');
-                  fs.access(oldDB, (err) => {
-                    if (!err) {
-                      fs.unlink(oldDB, (err1) => {
-                        if (err1) {
-                          // eslint-disable-next-line no-console
-                          console.log(`Could not delete old database: ${err1}`);
-                        }
+                  decompress(dbCompressed, userDataPath,
+                    {
+                      plugins: [
+                        decompressBzip2({ path: 'sttmdesktop.realm' }),
+                      ],
+                    })
+                    .then(() => {
+                      fs.chmodSync(dbPath, '755');
+                      module.exports.initDB();
+                      store.set('curDBHash', newestDBHash);
+                      fs.unlinkSync(dbCompressed);
+
+                      // Delete pre-5.0 DB
+                      const oldDBs = ['data.db', 'sttmdesktop.db'];
+                      oldDBs.forEach((oldDB) => {
+                        const oldDBPath = path.resolve(userDataPath, oldDB);
+                        fs.access(oldDBPath, (err) => {
+                          if (!err) {
+                            fs.unlink(oldDBPath, (err1) => {
+                              if (err1) {
+                                // eslint-disable-next-line no-console
+                                console.log(`Could not delete old database ${oldDB}: ${err1}`);
```

Comment:
should we some how let the user know to try and manually clean it up? I'm guessing it will just be taking up space on the user's machine.

Reply:
Nahee Jee. This is just a placeholder until we start doing proper logs and sending them back so we can find out about these kinds of things.